domains/Links/Tests/Feature/LinksStoreTest.php

```diff
@@ -84,4 +84,35 @@ public function it_fails_to_store_resources_on_validation_errors(): void
                 'tags',
             ]);
     }
+
+    /** @test */
+    public function it_fails_to_store_resources_with_invalid_link(): void
+    {
+
+        //assert link validation using simple string
+        $payload = [
+            'link' => "this_is_not_a_real_url",
```

Comment:
Can we make use of PHPUnit's data providers, here? It'll remove some duplicate code in this test.

Reply:
I tried using provider notation but I kept getting

The data provider specified for Domains\Links\Tests\Feature\LinksStoreTest::it_fails_to_store_resources_with_invalid_link is invalid.
InvalidArgumentException: Unknown formatter "name"

I believe it's due to that App is not bootstrapped yet as after setup.